Web/starter_project_iconics/components/about/CardItems.tsx

```diff
@@ -0,0 +1,16 @@
+import Image from "next/image"
+
+const Carditems = (props: any) => {
```

Comment:
Avoid using 'any' for types. If it has props, define the interface props and pass that to the function